src/components/ImageDetails/ImageDetails.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Input, Box, CloseButton } from '@chakra-ui/react';
+
+function ImageDetails({
+  index,
+  name,
+  preview,
+  // register,
+  defaultDescription,
+  removeImage,
+  removeDescription,
+  updateDescription,
+}) {
+  // let currentDescription;
+  // const changeDescription = e => {
+  //   updateDescription(index, e.target.value);
+  //   // updateDescription(index, e.target.value);
+  // };
+  const [currentDescription, setCurrentDescription] = useState();
+  // setFilesIntermediate(prev => {
+  //     prev[i].description = '';
+  //     return prev;
+  // })
+  return (
+    <Box>
```

Comment:
Since you guys merged in dev already and are using `@chakra-ui/react` ver 2.4.0, you can use `Card` components instead of `Box` for better default styling.